lib/net/ldap/error.rb

```diff
@@ -9,7 +9,22 @@ class Error < StandardError; end
 
   class AlreadyOpenedError < Error; end
   class SocketError < Error; end
-  class ConnectionRefusedError < Error; end
+  class ConnectionRefusedError < Error;
+    def initialize(*args)
+      warn_deprecation_message
+      super
+    end
+
+    def message
+      warn_deprecation_message
+      super
+    end
+
+    private
+    def warn_deprecation_message
+      warn "Deprecation warning: Net::LDAP::ConnectionRefused will be deprecated. Use Errno::ECONNREFUSED instead."
```

Comment:
When an user use this exception, this will show this exception will be removed. how about this? @jch